tests/MongoDB.Bson.Tests/Serialization/Conventions/ImmutableTypeClassMapConventionTests.cs

```diff
@@ -125,5 +143,25 @@ public void TestNoDefaultConstructorClassMapConventionWithTestClassC()
             Assert.True(classMap.HasCreatorMaps);
             Assert.Equal(1, classMap.CreatorMaps.Count());
         }
+                
+        [Fact]
+        public void TestNoDefaultConstructorClassMapConventionWithTestClassD()
+        {
+            var convention = new ImmutableTypeClassMapConvention();
+            var classMap = new BsonClassMap<TestClassD>();
+            convention.Apply(classMap);
+            Assert.True(classMap.HasCreatorMaps);
+            Assert.Equal(1, classMap.CreatorMaps.Count());
+        }
+
+        [Fact]
+        public void TestNoDefaultConstructorClassMapConventionWithTestClassE()
```

Comment:
this test is ok. 
But since the change from this ticket excluded the static properties. I expected the test for this class:

        private class TestClassF
        {
            public TestClassD(int a) { A = a; }

            public static TestClassD Default { get; set; } = new TestClassD(0);

            public int A { get; }
        }

So, I suggest adding one more test.